packages/form-js-viewer/test/spec/render/components/form-fields/Text.spec.js

```diff
@@ -108,6 +108,66 @@ Some _em_ **strong** [text](#text) \`code\`.
   });
 
 
+  it('should render markdown link (no external link)', function() {
+
+    // given
+    const text = '[forms](https://bpmn.io/)';
+
+    // when
+    const { container } = createText({
+      field: {
+        text,
+        type: 'Text'
+      },
+      properties: {
+        useTargetBlank: false
```

Comment:
Just discussing alternatives here: instead of having this as a simple flag, we could also make this a bit more generic, e.g.

```js
useTarget: '_blank',
useTarget: 'disabled',
useTarget: 'a custom behavior'
```

Reply:
That would probably make it more difficult to have both disabled links in the editor and blank links in the viewer, which could be solved via splitted configs in the playground (what we anyway need? cf. comment above).

Reply:
We'd need to consider whether this can cause XSS issues, so probably we are fine as is.

Reply:
According to https://www.w3schools.com/jsref/prop_anchor_target.asp, there are 5 types of targets:

_blank (new window), _self (default), _parent(outer frame), _top(outermost frame), and framename, a specific frame. 

I don't think it's an XSS issue because the control of this parameter is still in the hands of the library user. If they somehow pipe user input into that, it's on them to check that this won't cause issue IMO.

Reply:
So I'm for just letting library consumers specify it.